dev/FHIR3-0-2-MM202001-Cert/GGZ-2-0/_reference/resources/ggz-druguse-Cannabis.xml

```diff
@@ -90,7 +90,7 @@
       <coding>
         <system value="http://snomed.info/sct"/>
         <code value="398705004"/>
-        <display value="Cannabis (substance)"/>
+        <display value="Cannabis, wiet marihuana, hasj"/>

```

Comment:
Op termibologie.nictiz staat enkel Cannabis. De andere 3 staan als andere opties. Moet dat dan allemaal in 1 display waarde?

Reply:
Zie ook hier de [binding](https://simplifier.net/nictizstu3-zib2017/2.16.840.1.113883.2.4.3.11.60.40.2.7.4.1--20171231000000) met de zib.